src/odb/src/db/dbUtil.cpp

```diff
@@ -1626,4 +1626,158 @@ bool dbCreateNetUtil::createSingleVia(dbNet* net,
   return true;
 }
 
+static int snapToGrid (int number, dbTech *db_tech) {
+    // This will round "number" to a value that respects the manufacturing grid
+    // For example, if the manufacturing grid is 5 microns and number = 20333,
+    // the procedure will output 20335.
+    auto grid = db_tech->getManufacturingGrid();
+    return round(number / double(grid) * grid);
```

Comment:
This is incorrect, use:

return round(number / double(grid)) * grid;

Reply:
modified